docs/src/main/asciidoc/reactive-event-bus.adoc

```diff
@@ -88,6 +88,17 @@ void consumeBlocking(String message) {
     // Something blocking
 }
 ----
+
+Alternatively, you can annotate your method with `@RunOnWorkerThread`:
```

Comment:
Again, `ConsumeEvent#blocking()` is ignored if the annotation is present.

Reply:
Added a note in the documentation.